styles/message.mcss

```diff
@@ -100,6 +104,10 @@ Message {
     }
 
     div.meta {
+      display: flex;
+      flex-direction: column-reverse;
+      align-items: flex-end;
+      justify-content: flex-end;
```

Comment:
The reason for stacking them was just to mirror the name + time combo on the left. I didn't want to make any significant structural changes at this point as I don't know the app well enough and wanted to limit the scope of the PR. 

I'd really like to look at merging the like count with the like button in the actions bar, and adding an icon for it. Do you like the V fingers from patchwork-core? I'd like to see that ported over, but I'm happy to find other ones if no one else likes that.